content/creator/wearables-and-emotes/manage-collections/uploading-smart-wearables.md

```diff
@@ -0,0 +1,165 @@
+
+<img src="/images/wearables-and-emotes/uploading-smart-wearables/07_thumbnail.png" width="800" />
+{{< /hint >}}
+
+### **Update Video Showcase**
+
+You can update your video showcase by clicking on the camera icon and browsing your computer. **The video max file size is 4MB and max duration is 15 seconds**.
+
+<img src="/images/wearables-and-emotes/uploading-smart-wearables/06_video.png" width="400"/>
+
+### **Properties**
+
+Next to the thumbnail you're going to find the properties of your wearable, number of triangles of your model, number of materials and textures.
+
+<img src="/images/wearables-and-emotes/uploading-smart-wearables/08_properties.png" width="400"/>
+
+## **Uploading Mouth, Eyes and Eyebrows**
+
+The mouth, eyes and eyebrows category have a different behaviour in the editor because these are just .png files. To upload these just drag and drop the png file as a transparent image (256X256 pixels). Mouth is going to be automatically tinted by skin color, same for the eyebrows tinted by the hair color.
+
+{{< hint warning >}} If you want the asset to be masked, so a part of the mouth or eyebrows is not affected by the tinting, upload a zip file with both the png and the mask files. Remeber that the mask file should have a suffix "\_mask" in order to work. {{< /hint >}}
+
+<img src="/images/wearables-and-emotes/uploading-smart-wearables/02_mouth_wearable.png" width="300"/>
+
+After that uploading your wearables you will end up with a screen like this, that shows the items in your collection.
+
+<img src="/images/wearables-and-emotes/uploading-smart-wearables/11_list_wearables.png" width="900"/>
+
+## **Setting the Price of Your Smart Wearables**
+
+You can set the price of your wearable by clicking on **_Set Price_**. This can all be edited anytime, so don’t worry if you want to change it later on. Prices are set in MANA. Remember that when you mint wearables, they are minted directly on Matic/Polygon. When a user purchases your item, the transaction will be conducted in Matic/Polygon MANA.
+
+You could also **_Make it Free_**, which means that the price will be set as 0 MANA and the beneficiary address will be null. Know that making it free (primary sale) does not prevent it from being sold at any price as a secondary sale.
+
+Don’t forget to set the beneficiary address, which is the one that will receive the MANA from your sales. You can use any Ethereum address you like. To automatically fill in the address you are logged in with, click **_I’m the Beneficiary._**
+
+<img src="/images/wearables-and-emotes/uploading-smart-wearables/12_set_price.png" width="400"/>
+
+Save the price and you will be back to the list of wearables in your collection. When you click on the item, you will get its general info. Click on the button **_Preview in Editor_** to see it on the editor.
```

Comment:
Save the price, and you will be back on the list of Wearables in your collection. When you click on the item, you will get its general info. Click on the button **_Preview in Editor_** to see it in the editor.